.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     environment:
 
       # Set this to the upstream version tag to use/build
-      UPSTREAM_TAG: v1.6.3
+      UPSTREAM_TAG: v1.7.2-rc1
```

Comment:
Is this just for testing or do we want to use the release candidate in production?

Reply:
testing, will update once golden is released

Reply:
this image will be used for adapting our deployment of the AWS-CNI as there are a few changes